### PR TITLE
Us 10

### PR DESCRIPTION
**Pull request by MatthewFox97**
## Description
Please explain briefly the changes you made here.

## Checklist
- [ ] All manual security tests passing (Described here: https://github.com/Broom94/Klattice/wiki/Manual-Security-Testing)
- [ ] All unit tests passing 
- [ ] All UI tests passing
- [ ] Code coverage suitable
- [ ] WAVE toolbar run on new or updated screens
- [ ] Branch named {feature|hotfix|task}/{US-.*}
- [ ] If your pull request depends on any other, please link them
- [ ] Approval from Tech lead
- [ ] Approval from team member
- [ ] Commit messages are meaningful
- [ ] Target is green https://travis-ci.com/Broom94/Klattice/jobs/216924893 
- [ ] Add `DO NOT MERGE` if you want to postpone merge (in labels)

### CODE REVIEW

File: api/db.js [hunk 1]
```diff
@@ -67,6 +79,8 @@ exports.getResponsibilities = function(callback) {
   db.query(
     "SELECT responsibility_id, band_id, responsibility_description FROM responsibility;",
     function(err, rows) {
+       
+      console.log(rows[0]);
```
Comment (matthewdoreilly) [inline api/db.js]: Remove this

File: klattice/src/app/components/home/home.component.ts [hunk 2]
```diff
@@ -79,6 +83,8 @@ export class HomeComponent implements OnInit {
     });
   }
 
+
```
Comment (jamescrone) [inline klattice/src/app/components/home/home.component.ts]: can this whitespace be removed?

File: klattice/src/app/data.service.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import { Injectable } from "@angular/core";
 import { HttpClient, HttpHeaders } from "@angular/common/http";
 
+
```
Comment (jamescrone) [inline klattice/src/app/data.service.ts]: same here

File: api/db.js (hunk 1 above)
Comment (jamescrone) [inline api/db.js]: is this console log needed?

File: klattice/src/app/components/home/home.component.ts [hunk 4]
```diff
@@ -33,6 +33,10 @@ export class HomeComponent implements OnInit {
   bandID = -1;
   bandName: string;
 
+  capabilityLeadId = -1;
+  capabilityMessage = "NA";
+  capabilityLeadName = "Matthew Fox";
```
Comment (matthewdoreilly) [inline klattice/src/app/components/home/home.component.ts]: I don't think these are used anywhere, they can be deleted

File: klattice/src/app/components/home/home.component.ts (hunk 2 above)
Comment (MatthewFox97) [reply klattice/src/app/components/home/home.component.ts]: done

File: klattice/src/app/data.service.ts (hunk 3 above)
Comment (MatthewFox97) [reply klattice/src/app/data.service.ts]: done

File: api/db.js (hunk 1 above)
Comment (MatthewFox97) [reply api/db.js]: done

Comment (MatthewFox97) [reply api/db.js]: done

File: klattice/src/app/components/home/home.component.ts (hunk 4 above)
Comment (MatthewFox97) [reply klattice/src/app/components/home/home.component.ts]: done

**Pull request merged**